modules/mockupdd-editor/src/main/webapp/scripts/persistence/MockupAutosaveService.js

```diff
@@ -2,6 +2,33 @@
 
 var timeToReload = 5000; // in milliseconds.
 
+function getHtmlToPersist () {
+  var $page = $('#page').clone();
+  $page.find('.ui-wrapper').each(function(i, e) {
+    var $parent = $(e).parent();
+    var $wrapper = $(e);
+    $(e).children().each(function(i, e) {
+      if (!$(e).is('div')) {
+        $(e).css('width', $wrapper.css('width'));
+        $(e).css('height', $wrapper.css('height'));
+        $parent.append($(e).clone());
+      }
+    });
+    $(e).remove();
+  });
+  $page.find('.ui-resizable').each(function(i, e) {
+    $(e).find('*').each(function(i, e) {
+      $(e).remove();
+    });
+  });
+  $page.find('*').each(function(i, e) {
+    $(e).attr('class', function(i, c){
+      return c.replace(/(^|\s)ui-\S+/g, '');
```

Comment:
Please add some comment in the code when you add regex, otherwise is hard to undestand